vision/detect/detect.go

```diff
@@ -14,22 +14,24 @@ import (
 	"io"
 	"os"
 
-	"cloud.google.com/go/vision"
+	"cloud.google.com/go/vision/apiv1"
```

Comment:
The type names are different, goimports will match cloud.google.com/go/vision/apiv1. I am suggesting to keep it this way.